server/clip_server/executors/clip_hg.py

```diff
@@ -181,18 +188,25 @@ async def encode(self, docs: DocumentArray, **kwargs):
             the CLIP model was trained on images of the size ``224 x 224``, and that
             they are of the shape ``[3, H, W]``  with ``dtype=float32``. They should
             also be normalized (values between 0 and 1).
+        :param parameters: A dictionary that contains parameters to control encoding.
+            The accepted keys are ``traversal_paths`` and ``minibatch_size`` - in their
+            absence their corresponding default values are used.
         """
+
+        traversal_paths = parameters.get('traversal_paths', self._traversal_paths)
```

Comment:
```suggestion
        traversal_paths = parameters.get('traversal_paths', '@r')
```

Reply:
I don't agree with this suggestion. I will break the following use case:

```
gateway -> encoder #1 (work on root_level) -> encoder #2(work on chunk_level)
```

It is impossible to pass the proper parameters:
```
client.post(on='/', parameters={'traversal_paths': '?????'})
```

Reply:
By defining the default traversal path in `__init__`, `client.pos(on='/')` works
```
gateway -> encoder #1 (traversal_paths='@r') -> encoder #2(traversal_paths='@c')
```

Reply:
`client.post(on='/', parameters={'traversal_paths': '?????'})`

We do not allow this anyway

Reply:
why it is impossible? i dont get it

Reply:
@hanxiao From the above flow example, there are two encoders both working on different level documents (one on root-level, another on chunk-level). 
- use `@r` on request parameter at client: `client.post(on='/', parameters={'traversal_paths': '@r'})`
-> encoder 2 cannot work 
- use `@c` on request parameter at client: `client.post(on='/', parameters={'traversal_paths': '@c'})`
-> encoder 1 cannot work

Reply:
but you should be able to send parameter to one particular Executor